Monika After Story/game/script-topics.rpy

```diff
@@ -7591,7 +7591,91 @@ label monika_cartravel:
     m 1hua "Just the idea of travelling around the world makes me so excited!"
     m 1eka "I really can't wait, [player]~"
     return
-
+    
+    ###poems archives
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_old_poem",
+            category=['literature'],
+            prompt="Old poems",
+            random=True,
+            aff_range=(mas_aff.AFFECTIONATE,None)
+        )
+    )
+    
+label monika_old_poem:
+    m 1eub "You know, [player]..."
+    m 1hub "I kept all the poems that I wrote for you!"
+    m 1dua "All the sheets are in a small jewel box{w}, since they are like a treasure for me.{w}"
+    m 1hua "So if you want to see one, I'll be pleased to help you!"
+    $ mas_unlockEVL("monika_show_poem","EVE")
+    return "derandom"
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_show_poem",
+            category=['literature'],
+            prompt="Can I see a poem?",
+            pool=True,
+            unlocked=True, #Unlocked for test
+            rules={"no unlock": None}
+        )
+    )
+label monika_show_poem:
+    if mas_isMoniUpset(lower=True):
```

Comment:
Instead of this, I'd just use the `aff_range` property in the ev.

Reply:
Done